pkg/datastore/postgres/errors.go

```diff
@@ -3,11 +3,14 @@ package postgres
 import (
 	"errors"
 	"fmt"
+
+	"github.com/lib/pq"
 )
 
 const (
-	DbConnectionFailedMsg = "db connection failed"
-	SettingNodeFailedMsg  = "unable to set db node"
+	DbConnectionFailedMsg        = "db connection failed"
+	SettingNodeFailedMsg         = "unable to set db node"
+    ForeignKeyViolationErrorCode = "23503"
```

Comment:
Missing `go fmt`? Weird that it missed this one.

Reply:
Nice catch!